library/common/BUILD

```diff
@@ -35,7 +35,15 @@ envoy_cc_library(
     deps = [
         "@envoy//source/common/common:random_generator_lib",
         "@envoy//source/common/runtime:runtime_lib",
-        "@envoy//source/exe:envoy_common_lib",
+        "@envoy//source/exe:main_common_lib",
+    ],
+)
+
+envoy_cc_library(
+    name = "envoy_mobile_main_common_lib_stamped",
+    repository = "@envoy",
+    deps = [
+        ":envoy_mobile_main_common_lib",
```

Comment:
Why is this now required?

Reply:
https://github.com/envoyproxy/envoy/pull/12171 split things for tests, so our targets here were either: a) not having symbols, or having duplicate symbols. This new setup lets us use the new upstream system correctly.